lib-src/portmixer/src/px_unix_oss.c

```diff
@@ -163,15 +172,15 @@ int OpenMixer_Unix_OSS(px_mixer *Px, int index)
    info->playback.num = 0;   
    
    do {
-      info->capture.name = PaOSS_GetStreamInputDevice(Px->pa_stream);
+      info->capture.name = GetDeviceName(Px->input_device_index);
```

Comment:
I notice these are the only non-Windows usages of the new fields `input_device_index` and `output_device_index`.  They happen only in non-Apple Unices.

So what about the case of Apple?  Is it an omission that the new fields are set but not used anywhere?  Or can you reassure me you studied the problem and decided there was no need?

Reply:
I do observe now that portaudio.patch never needed anything specific to macOS, so that persuades me it's not an omission.